contents/newsletter/good-taste-great-products.md

```diff
@@ -0,0 +1,209 @@
+
+The more you do this, the more you’ll instinctively build them into your work, thus making your software feel effortless and familiar. 
+
+> **Recommended reading:** [_Design is Storytelling_](https://www.amazon.com/Design-Storytelling-Ellen-Lupton/dp/194230319X) by Ellen Upton is a fun treatise on the power of narrative to help software communicate with users.
+
+* * *
+
+##  **2. Detect and eliminate obvious jank**
+
+You don’t need to be the most talented designer on earth to have taste.
+
+Instead, you need a reliable sense of what is janky and what is nice, and the ability to nudge a product in the nicer direction.
+
+People who encounter janky software treat it with suspicion, and eventually disdain. They lose trust in the people building it, and won’t recommend it.
+
+In contrast, pleasant software suggests care and attention, and earns loyalty. Beneath the surface, it is assumed, yet more care is lurking. Decently designed products feel more trustworthy.
+
+You can spend a lifetime learning design, but you can learn how to spot jankiness right now.
+
+###  **Make friends with the grid**
+
+Grids make things look tidy and deliberate. If something feels _vaguely janky,_ it might be because visual elements aren’t lining up according to any rules. 
+
+![](https://res.cloudinary.com/dmukukwp6/image/upload/3a940cd7_5227_472c_bf55_d47193d8010d_2020x962_40f183c715.png)
+
+Scattered elements laying everywhere are janky. Even aligning things by their leading or trailing edges can get you pretty far out of jank debt.
+
+###  **Be disciplined in your typography**
+
+![](https://res.cloudinary.com/dmukukwp6/image/upload/6119a619_3e6a_47c8_a218_b525db48f8cc_1458x706_edb5d2b620.png)
+
+Instead of going wild with multiple typefaces, try using a single typeface with variations in weight to draw the eye towards what’s most important. This creates hierarchy, making your interface easier to scan.
+
+![](https://res.cloudinary.com/dmukukwp6/image/upload/6d3d0326_8242_4cd1_9a15_63c5018ee3d0_1460x750_555ba64c95.png)
+
+>  **Recommended reading:** Typography is a huge subject, [but here’s a great start](https://www.interaction-design.org/literature/topics/typography?srsltid=AfmBOoqjW0UJxE_dKkPai6HgXGrHXxSicAlWEu2AFIP3PPtVQNLwuggc) on the basics from the Interaction Design Foundation. [Typography in ten minutes](https://practicaltypography.com/typography-in-ten-minutes.html) from Practical Typography is an excellent read, too.
+
+###  **Mind your hierarchy**
+
+Speaking of hierarchy, where we position things communicates their importance. If something is essential for your users to see or interact with, don’t bury it.
+
+In Stellaris, a strategy game about space exploration and expansion, resources like energy production, minerals, and food are central to every decision, so the UI places them at the top of the screen, visible at all times.
+
+![](https://res.cloudinary.com/dmukukwp6/image/upload/bd7f1a9b_c1c3_40da_9a78_3e233f9326d5_2204x1920_3f630c99b6.png)
```

Comment:
Same. In dark mode, you can't see the text on this image sadly. Maybe needs a separate version